geode-core/src/test/java/org/apache/geode/internal/cache/tier/sockets/command/Put70Test.java

```diff
@@ -328,14 +325,14 @@ void shouldSetPossibleDuplicateReturnsTrueIfNotRecoveredVersionTagAndWithPersist
 
   @Test
   void isRegionWithPersistenceReturnsTrueIfDataPolicyWithPersistence() {
-    when(dataPolicy.withPersistence()).thenReturn(true);
+    when(attributes.getDataPolicy()).thenReturn(DataPolicy.PERSISTENT_REPLICATE);
 
     assertThat(put70.isRegionWithPersistence(localRegion)).isTrue();
   }
 
   @Test
   void isRegionWithPersistenceReturnsTrueIfIsAccessorAndHavingPersistentMembers() {
-    when(dataPolicy.withPersistence()).thenReturn(false);
+    when(attributes.getDataPolicy()).thenReturn(DataPolicy.PARTITION);
```

Comment:
The older code seems to be looking to know if its configured with persistent data-policy.
The new change is different than that. Is it what is expected in the test.
In the previous above change; for the same old code its returning DataPolicy.PERSISTENT_REPLICATE.
Same comment with following two changes in this file.

Reply:
logically it's the same here.

Reply:
You fill find lots of places in the old tests where the combinations of mocks and return values used for `DataPolicy` actually created mutually exclusive combinations but it didn't matter for the sake of the test. For the changes here I tried to match up what the test was actually expecting for the correct combinations that aligned to an enum value.

Reply:
Based on the test name; is it expecting it to treat as persistent when it is accessor but not configured with persistence and has persistent members?
If thats the case; the change seems to be defeating the purpose by not having:
when(dataPolicy.withPersistence()).thenReturn(false);